packages/management/tanzu-framework-manager/metadata.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  name: tanzu-framework-manager.tanzu.vmware.com
```

Comment:
Should we name this `framework.tanzu.vmware.com`? It avoids `tanzu-xxx.tanzu` stutter and I think `manager` is unnecessary. I'd like some other opinions.

cc @jmoroski @vijaykatam @vuil

Reply:
`framework` is a little overloaded term, so I'm a little hesitant in using that as the name of the package, but if you have a strong disagreement with `tanzu-framework-manager`, we can think of other names.

Reply:
tanzu is redundant since domain already has it and manager typically implies a controller. This is a meta package based on recent discussions so I am in favor of just framework and maybe check what TAP is doing with naming conventions for meta and try to see if we can try to align. 

Related change https://github.com/vmware-tanzu/community-edition/pull/3568/files

Reply:
@yharish991 `framework.tanzu.vmware.com` is just a canonical name for `tanzu-framework` and conveys the same meaning. I don't really think it's overloaded.